issues/gnqa_integration_to_global_search_Design.gmi

```diff
@@ -0,0 +1,56 @@
+# GNQA Integration to Global Search Design Proposal
+
+## Tags
+* assigned: jnduli, alexm
+* keywords: llm, genenetwork2
+* type: feature
+* status: in-progress
+
+## Description
+This document outlines the design proposal for integrating GNQA into the Global Search feature.
+
+## High-Level Design
+
+### UI Design
+When the GN2 Global Search page loads:
+1. A request is initiated via HTMX to the GNQA search page with the search query.
+2. Based on the results, a page or subsection is rendered, displaying the query and the answer, and providing links to references.
+
+For more details on the UI design, refer to the pull request: 
+=> https://github.com/genenetwork/genenetwork2/pull/862
+
+### Backend Design
+The API handles requests to the Fahamu API and manages result caching. Once a request to the Fahamu API is successful, the results are cached using SQLite for future queries. Additionally, a separate API is provided to query cached results.
+
+## Deep Dive
+
+### Caching Implementation
+For caching, we will use SQLite3 since it is already implemented for search history. Based on our study, this approach will require minimal space:
+
+*Statistical Estimation:*  
+We calculated that this caching solution would require approximately 79MB annually for an estimated 20 users, each querying the system 5 times a day.
```

Comment:
More details would help here. My suggestion would be to add:

1. average request size per user and how we determined this
2. why we're assuming 20 users making 5 requests per day

Reply:
The average request size was an upper bound calculation for documents returned from the Fahamu API. We’re assuming 20 users making 5 requests per day to estimate typical usage of GN2 services.

Reply:
PS:  Apologies,  Got your question wrong, But this is now captured in the docs.